flake.nix

```diff
@@ -6,12 +6,20 @@
 
   outputs = { self, nixpkgs, flake-utils }:
     flake-utils.lib.eachDefaultSystem (system:
-      let pkgs = nixpkgs.legacyPackages.${system};
+      let
+        pkgs = nixpkgs.legacyPackages.${system};
       in {
         devShell = pkgs.mkShell {
-          nativeBuildInputs = [
-            pkgs.nodejs-16_x
+          nativeBuildInputs = with pkgs; [
+            nodejs-18_x
+            nodePackages.pnpm
+            turbo
           ];
+
+          shellHook = ''
+            export PATH="$PWD/node_modules/.bin/:$PATH"
```

Comment:
Is there any reason to add `node_modules` here?

Reply:
We can remove it, I think it's a convention I used since sometimes scripts in the bin folder weren't loaded, but maybe that's not needed anymore.